src/core/Microsoft.PowerToys.Settings.UI.Lib/PowerRenameSettings.cs

```diff
@@ -18,6 +18,19 @@ public PowerRenameSettings()
             name = "_unset_";
         }
 
+        public PowerRenameSettings(PowerRenameLocalProperties localProperties)
+        {
+            properties = new PowerRenameProperties();
+            properties.PersistState.Value = localProperties.PersistState;
+            properties.MRUEnabled.Value = localProperties.MRUEnabled;
+            properties.MaxMRUSize.Value = localProperties.MaxMRUSize;
+            properties.ShowIcon.Value = localProperties.ShowIcon;
+            properties.ExtendedContextMenuOnly.Value = localProperties.ExtendedContextMenuOnly;
+
+            version = "1";
+            name = "_unset_";
```

Comment:
I think you will need to set the module name here.

Reply:
yes since we already know it.